flax/jax_utils.py

```diff
@@ -72,6 +72,8 @@ def replicate(tree, devices=None):
   Returns:
     A new pytree containing the replicated arrays.
   """
+  warnings.warn('use jax.device_put_replicated',
```

Comment:
Could you add a deprecation warning to the docstring as well? Similar to e.g. here: https://github.com/google/flax/blob/master/flax/nn/attention.py#L48